Most my tests worked, but I found two cases that fail:
- When selecting an asset type (for instance in our test prod if I select `/Assets/BG_Harmony` or `/Assets/BG_Photoshop` it fails and I get this error:
```
==============================
'tuple' object has no attribute 'replace'
==============================
Traceback (most recent call last):
  File "/Users/normaal/Library/Application Support/AYON/addons/core_1.6.13+normaal.0.1/ayon_core/tools/launcher/models/actions.py", line 191, in trigger_action
    action.process(selection)
  File "/Users/normaal/dev/repos/ayon-kitsu/client/ayon_kitsu/plugins/launcher/launcher_show_in_kitsu.py", line 40, in process
    webbrowser.open(
  File "webbrowser.py", line 86, in open
  File "webbrowser.py", line 670, in open
AttributeERROR: 'tuple' object has no attribute 'replace'
```
- There's an empty folder at the root in our test prod that gives a 404 in kitsu. Which is probably normal actually, it most likely just doesn't correspond to anything in Kitsu hence the 404.

--- a/client/ayon_kitsu/plugins/launcher/launcher_show_in_kitsu.py
+++ b/client/ayon_kitsu/plugins/launcher/launcher_show_in_kitsu.py
@@ -126,8 +126,8 @@ class ShowInKitsu(LauncherAction):
             gazu.project.get_project_url({"id": project_kitsu_id})
         )
         return (
-            f"{project_url.parent}/episodes/all/assets",
-            f"?search=+type=[{folder_label}]",
+            f"{project_url.parent}/episodes/all/assets"
+            f"?search=+type=[{folder_label}]"
         )
 
     def _get_sequence_url(

